cpp/include/cuvs/neighbors/cagra.h

```diff
@@ -73,6 +73,61 @@ cuvsError_t cuvsCagraIndexParamsCreate(cuvsCagraIndexParams_t* params);
  */
 cuvsError_t cuvsCagraIndexParamsDestroy(cuvsCagraIndexParams_t params);
 
+/** Parameters for VPQ compression. */
+struct cuvsCagraCompressionParams {
+  /**
+   * The bit length of the vector element after compression by PQ.
+   *
+   * Possible values: [4, 5, 6, 7, 8].
+   *
+   * Hint: the smaller the 'pq_bits', the smaller the index size and the better the search
+   * performance, but the lower the recall.
+   */
+  uint32_t pq_bits = 8;
```

Comment:
In class member initializers aren't valid in C (only are in c++11) . This has previously caused problems trying to compile with a C compiler :  https://github.com/rapidsai/cuvs/pull/28 .

We should probably just remove these default values ,  since the default values are going to get picked up from the c++ struct in `cuvsCagraCompressionParamsCreate`

Reply:
Thanks, Ben! I totally missed this in the copy-paste process